src/appfl/algorithm/fedavg.py

```diff
@@ -69,8 +69,8 @@ def update(self):
                 target = target.to(self.device)
 
                 optimizer.zero_grad()
-                output = self.model(data)
-                loss = self.loss_fn(output, target)
+                output = self.model(data)                
+                loss = self.loss_fn(output, target)                                                
```

Comment:
unnecessary white spaces... can you run black formatting? maybe for all the files that are updated?